tools/psa/release.py

```diff
@@ -74,7 +74,14 @@ def _get_target_info(target):
     if not os.path.exists(delivery_dir):
         raise Exception("{} does not have delivery_dir".format(target))
 
-    return tuple([TARGET_MAP[target].name,
+    if toolchain:
+        if toolchain not in TARGET_MAP[target].supported_toolchains:
+            raise Exception("Toolchain {} was not found in PATH".format(toolchain))
```

Comment:
```suggestion
            raise Exception("Toolchain {} is not supported by {}".format(toolchain, TARGET_MAP[target].name))
```

Error message should change as well

Reply:
done